source/_integrations/goalzero.markdown

```diff
@@ -7,18 +7,30 @@ ha_iot_class: Local Polling
 ha_release: 0.116
 ha_config_flow: true
 ha_domain: goalzero
-ha_codeowners:
-  - '@tkdrob'
 ha_platforms:
   - binary_sensor
+  - switch
+ha_codeowners:
+  - '@tkdrob'
 ---
 
 This `goalzero` integration pulls data from a Wifi enabled Goal Zero Yeti.
 
-{% include integrations/config_flow.md %}
+## Configuration
+
+Go to the integrations page in your configuration and click on new integration -> Goal Zero Yeti.
 
 ## Integration Entities
 
-Each added configuration entry will create the following sensors:
+Each added configuration entry will create the following binary sensors:
+
+- `backlight`
+- `app_online`
+- `charging`
+- `input_detected`
```

Comment:
It is generally better to provide a description of the added sensors. These seem like entity IDs, which are not guaranteed.